pkg/machines/components/vmnetworktab.jsx

```diff
@@ -23,12 +23,30 @@ import { changeNetworkState } from "../actions/provider-actions.es6";
 import VmLastMessage from './vmLastMessage.jsx';
 import { Listing, ListingRow } from 'cockpit-components-listing.jsx';
 import { rephraseUI, vmId } from "../helpers.es6";
+import EditNICAction from './nicEdit.jsx';
+import { Tooltip } from 'cockpit-components-tooltip.jsx';
+import './nicEdit.css';
 
 const _ = cockpit.gettext;
 
-const VmNetworkTab = function ({ vm, dispatch, hostDevices }) {
+const VmNetworkTab = function ({ vm, dispatch, config, hostDevices, networks }) {
     const id = vmId(vm.name);
 
+    const warningInactive = (id) => {
+        return (
+            <Tooltip tip={_("Changes will apply on VM restart")} pos='top'>
+                <span id={id} className='fa-stack fa-sm'>
+                    <i className='fa fa-circle-thin fa-stack-2x fa-stack-xs' />
```

Comment:
Shouldn't this also be `pf-pending`?

Reply:
Right, as the other one, I forgot to change this one. Thanks